appveyor.yml

```diff
@@ -0,0 +1,36 @@
+environment:
+  matrix:
+    - nodejs_version: 4
+      platform: x64
+      msvs_toolset: 12
+    - nodejs_version: 6
+      platform: x64
+      msvs_toolset: 12
+    - nodejs_version: 7
+      platform: x64
+      msvs_toolset: 12
+    - nodejs_version: 8
+      platform: x64
+      msvs_toolset: 12
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - npm install
+
+# Post-install test scripts.
+test_script:
```

Comment:
Tests are not being executed right now.
I think this should be `test:` instead `test_script:`